RockPaperScissors/RockPaperScissors.cs

```diff
@@ -4,22 +4,91 @@ namespace RockPaperScissors
 {
     class Program
     {
+        public static int pScore = 0;
+        public static int cScore = 0;
+
         public static void Main()
         {
-            Console.WriteLine("Enter hand 1:");
-            string hand1 = Console.ReadLine().ToLower();
-            Console.WriteLine("Enter hand 2:");
-            string hand2 = Console.ReadLine().ToLower();
-            Console.WriteLine(CompareHands(hand1, hand2));
+            //the user is allowed a choice of hand to play with
+            Console.WriteLine("Do you choose Rock, Paper, or Scissors?");
+            string pHand = Console.ReadLine().ToLower();
+            string cHand = System.String.Empty;
+
+//generate random number and assign to an integer
+            Random rng = new Random();
+            int ran = rng.Next(0, 2);
```

Comment:
This will only choose a random number of 0 or 1.
https://msdn.microsoft.com/en-us/library/2dx6wyd4(v=vs.110).aspx